tensorflow/stream_executor/cuda/cublas_11_0.inc

```diff
@@ -5030,11 +5194,4 @@ void CUBLASWINAPI cublasZtrmm(char side, char uplo, char transa, char diag,
   return func_ptr(side, uplo, transa, diag, m, n, alpha, A, lda, B, ldb);
 }
 
-CUBLASAPI const char* CUBLASWINAPI cublasGetStatusString(cublasStatus_t status) {
```

Comment:
Looks like this was added after the PR was created. Is the only issue now the fact that the `cublas_plugin` rule was linked into two shared libraries?

Reply:
Yes, that's correct.
It looks like the `cublasGetStatusString` stub was added ad-hoc while I used the script to regenerate the whole file. Let me know if you want me to remove my changes to this file.

Reply:
I think the changes are fine. Better to have these changes than not have them.